modules/ROOT/pages/migration/index.adoc

```diff
@@ -20,6 +20,151 @@ npm update @neo4j/graphql
 
 Here is a list of all the breaking changes from version 5.0.0 to 6.0.0.
 
+=== Changed minimum Node.js version to 20.0.0
+
+The minimum Node.js version required to run the Neo4j GraphQL Library is now 20.0.0.
+
+=== Changed minimum Neo4j version to 5.0.0
+
+The minimum Neo4j version required to run the Neo4j GraphQL Library is now 5.0.0.
+
+=== Removed the deprecated `@relationship` implicit `_SOME` filters
+
+The deprecated `@relationship` filter without the operator suffix has been removed in favor of the `_SOME` filter.
+
+[cols="1,1"]
+|===
+|Before | Now
+
+a|
+[source, graphql, indent=0]
+----
+{
+  movies(where: { actors: { name: "Keanu" } }) {
+    title
+  }
+}
+----
+a|
+[source, graphql, indent=0]
+----
+{
+  movies(where: { actors_SOME: { name: "Keanu" } }) {
+    title
+  }
+}
+----
+|===
+
+=== Invalid filters in Aggregation filter inputs are being removed
+
+In previous versions of the Library, the input used for aggregation filters contained filters that did not rely on aggregating functions. Those filters were deprecated in previous Library versions and removed from the schema.
+
+To see the aggregating functions supported on filtering refer to xref:/queries-aggregations/filtering.adoc#_aggregation_filtering[Aggregation filtering],
+
+=== String aggregation filters that are not using `_LENGTH` are removed
+
+String aggregations are computed using lengths, and the `_LENGTH` suffix is now required for all string aggregation filters.
+
+[cols="1,1"]
+|===
+|Before | Now
+
+a|
+[source, graphql, indent=0]
+----
+{
+  movies(
+    where: { actorsAggregate: { node: { name_AVERAGE_EQUAL: 1.5 } } }
+  ) {
+    title
+  }
+}
+
+----
+a|
+[source, graphql, indent=0]
+----
+{
+  movies(
+    where: { actorsAggregate: { node: { name_AVERAGE_LENGTH_EQUAL: 1.5 } } }
+  ) {
+    title
+  }
+}
+
+----
+|===
+
+=== Removed the deprecated `options` argument from `assertIndexesAndConstraints`
+
+The deprecated `options` argument from the `assertIndexesAndConstraints` utility has been removed.
+Database migrations are outside of the scope of the Neo4j GraphQL Library, and all indexes and constraints will have to be managed manually.
```

Comment:
```suggestion
Database migrations are outside of the scope of the Neo4j GraphQL Library, and all indexes and constraints will have to be managed manually.
```

Worth mentioning that we still provide support for verifying? Also potentially linking to documentation about how to create indexes, just to be nice?